dev/tests/integration/testsuite/Magento/Developer/Console/Command/DiInfoCommandTest.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
```

Comment:
Please add `declare(strict_types=1);` in accordance of [technical guidelines](https://developer.adobe.com/commerce/php/coding-standards/technical-guidelines/#1-basic-programming-principles)
```suggestion
declare(strict_types=1);
```

Reply:
Thanks, I will make the required edits and correct the tests a little later on